azext_edge/tests/edge/support/test_support_unit.py

```diff
@@ -562,6 +571,65 @@ def test_get_bundle_path(mocked_os_makedirs):
     assert str(path).startswith(expected) and str(path).endswith("_aio.zip")
 
 
+# TODO - test zipfile write for specific resources
+# MQ connector stateful sets need labels based on connector names
+@pytest.mark.parametrize(
+    "mocked_cluster_resources",
+    [[MQ_API_V1B1]],
+    indirect=True,
+)
+@pytest.mark.parametrize(
+    "custom_objects",
+    [
+        # connectors present
+        {
+            "items": [
+                {
+                    "metadata": {"name": "mock-connector", "namespace": "mock-namespace"},
+                },
+                {
+                    "metadata": {"name": "mock-connector2", "namespace": "mock-namespace2"},
+                },
+            ]
+        },
+        # no connectors
+        {"items": []},
+    ],
+)
+def test_mq_list_stateful_sets(
```

Comment:
Is this intended to be a temporary/one-off  test ?

Reply:
It feels like the entire support bundle test needs to be refactored already... so not sure how much of a difference would this being a one-off create

Reply:
It is in the sense that I didn't have a good way to overload the basic support bundle test that assumes each provider only calls `list_stateful_sets_in_all_namespaces` - since MQ now has to call again with `field_selectors`, (which, also theoretically temporary since I think these resources SHOULD have an MQ label) I wanted to ensure it was being called correctly depending on the particular connector resources.